ITEM_en.tsv

```diff
@@ -4199,104 +4199,104 @@ ITEM_20150406_004575	A plow for ploughing fields. It has a keen blade.
 ITEM_20150406_004576	A spade for clearing weeds. It has a sharp point.
 ITEM_20150406_004577	Farm equipment for making furrows. It has a sharp point.
 ITEM_20150406_004578	Farm equipment for removing stones. It is well made.
-ITEM_20150406_004579	Well dried beef jerky.
+ITEM_20150406_004579	Farm equipment for separating fields. It has a lot of sharp points.
 ITEM_20150406_004580	Fresh Meat
-ITEM_20150406_004581	Magic stone of blessing
-ITEM_20150406_004582	Wing of orange Dandel.
-ITEM_20150406_004583	Wing of orange Dandel. It does seem to be very burn
-ITEM_20150406_004584	Research material of Varkis (1)
-ITEM_20150406_004585	Research material of Varkis
-ITEM_20150406_004586	Research material of Varkis (2)
-ITEM_20150406_004587	Research material of Varkis (3)
-ITEM_20150406_004588	Research material of Varkis (4)
+ITEM_20150406_004581	Magical Stone of Blessing
+ITEM_20150406_004582	Orange Dandel Wingpice
+ITEM_20150406_004583	Wing of an Orange Dandel. Seems flammable.
+ITEM_20150406_004584	Varkis' Research Material (1)
+ITEM_20150406_004585	Varkis' Research Material
+ITEM_20150406_004586	Varkis' Research Material (2)
+ITEM_20150406_004587	Varkis' Research Material (3)
+ITEM_20150406_004588	Varkis' Research Material (4)
 ITEM_20150406_004589	Lizardman Toenail
-ITEM_20150406_004591	Tama Body Fluids
-ITEM_20150406_004596	Root of young vine
-ITEM_20150406_004598	Horn's piece of Short tree
+ITEM_20150406_004591	Tama Body Fluid
+ITEM_20150406_004596	Young Grapevine Root
+ITEM_20150406_004598	Short Tree Root Fragment
 ITEM_20150414_004603	Tag Bow
 ITEM_20150414_004604	Accuracy Crossbow
-ITEM_20150414_004605	Hammer
-ITEM_20150414_004606	Grang Spontune
-ITEM_20150414_004607	Dagger (Sting)
+ITEM_20150414_004605	Toy Hammer
+ITEM_20150414_004606	Grand Spontoon
+ITEM_20150414_004607	Dagger (Pierce)
 ITEM_20150414_004608	Superior Knife
 ITEM_20150414_004609	Superior Dirk Dagger
 ITEM_20150414_004610	Superior Main Gauche
-ITEM_20150414_004611	Pistol (Attack)
+ITEM_20150414_004611	Pistol (Shot)
 ITEM_20150414_004612	Cotton Gloves
 ITEM_20150414_004613	Cotton Boots
 ITEM_20150414_004614	Cotton Pants
-ITEM_20150414_004615	Cotton Robe
-ITEM_20150414_004616	Don't be deliberately damaged even you want to check the magic defense.
-ITEM_20150414_004617	Gloves Keeper
-ITEM_20150414_004618	Equipment used by Keeper
-ITEM_20150414_004619	Boots Keeper
-ITEM_20150414_004620	Pants Keeper
-ITEM_20150414_004621	Tunic Keeper
+ITEM_20150414_004615	Cotton Robes
+ITEM_20150414_004616	Don't test this armor's Magic Defense by intentionally being hit.
+ITEM_20150414_004617	Watchman Gloves
+ITEM_20150414_004618	Equipment used by Watchmen.
```

Comment:
Watchmen -> watchmen

I'm assuming these are just generic guards/watchmen, unless they need to be capitalized here.